test/k8sT/Services.go

```diff
@@ -412,6 +412,93 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	Context("Checks local redirect policy", func() {
+		var (
+			deploymentYAML string
+			lrpServiceName string
+			app1IP         string
+			app2IP         string
+			be1IP          string
+			be2IP          string
+		)
+
+		BeforeAll(func() {
+			deploymentYAML = helpers.ManifestGet(kubectl.BasePath(), "lrp-test.yaml")
+			lrpServiceName = "lrp-demo-service"
+			res := kubectl.ApplyDefault(deploymentYAML)
+			res.ExpectSuccess("Unable to apply %s", deploymentYAML)
+			waitPodsDs()
+		})
+
+		AfterAll(func() {
+			_ = kubectl.Delete(deploymentYAML)
+		})
+
+		getSingletonPodIP := func(label string) string {
+			podsIPs, err := kubectl.GetPodsIPs(helpers.DefaultNamespace, label)
+			Expect(err).To(BeNil(), "Cannot get pod IP for %s pod", label)
+			Expect(len(podsIPs)).To(Equal(1), "Got more than 1 ip for %s pod", label)
+			for _, ip := range podsIPs {
+				return ip
+			}
+			// Should never reach here
+			return ""
```

Comment:
Maybe then it makes sense to add a failing assertion instead of returning the empty string?

Reply:
I think the assertion is covered before the loop above, this is just to make compiler happy, but maybe it's a bad practice?